acquire/acquire.py

```diff
@@ -1217,9 +1217,13 @@ class RemoteAccess(Module):
         ("glob", "sysvol/Program Files/TeamViewer/*.log"),
         ("glob", "sysvol/Program Files (x86)/TeamViewer/*.log"),
         ("glob", "AppData/Roaming/TeamViewer/*.log", from_user_home),
-        # anydesk
+        # anydesk - Windows
         ("dir", "sysvol/ProgramData/AnyDesk"),
         ("glob", "AppData/Roaming/AnyDesk/*.trace", from_user_home),
+        ("glob", "AppData/Roaming/AnyDesk/*/*.trace", from_user_home),
```

Comment:
```suggestion
        ("glob", "AppData/Roaming/AnyDesk/**/*.trace", from_user_home),
```
Would this work?

Reply:
It would work, but I don't know if it will negatively impact the runtime, since it will then traverse all of the paths